src/main/java/org/opentripplanner/index/model/TripTimeShort.java

```diff
@@ -1,21 +1,23 @@
 package org.opentripplanner.index.model;
 
-import java.util.List;
-
 import org.opentripplanner.model.FeedScopedId;
 import org.opentripplanner.model.Stop;
+import org.opentripplanner.model.StopTime;
 import org.opentripplanner.model.Trip;
 import org.opentripplanner.routing.core.ServiceDay;
 import org.opentripplanner.routing.edgetype.Timetable;
 import org.opentripplanner.routing.trippattern.RealTimeState;
 import org.opentripplanner.routing.trippattern.TripTimes;
 
-import com.beust.jcommander.internal.Lists;
+import java.util.ArrayList;
+import java.util.List;
+
 
 public class TripTimeShort {
 
     public static final int UNDEFINED = -1;
     public FeedScopedId stopId;
+    public StopTime stopTime;
```

Comment:
This class is part of the OTP REST API - unsure if this will work with all clients. Also, I do not understand if this needs to be changed, we can probably ONLY change the GraphQL endpoint.